cmd/soroban-rpc/internal/transactions/transactions_test.go

```diff
@@ -311,6 +347,21 @@ func TestIngestTransactions(t *testing.T) {
 	require.Len(t, store.transactions, 3)
 }
 
+func TestGetTransactionsWithEventData(t *testing.T) {
+	store := NewMemoryStore(interfaces.MakeNoOpDeamon(), "passphrase", 100)
+
+	// Insert ledger 1
+	metaWithEvents := txMetaWithEvents(1, false)
+	require.NoError(t, store.IngestTransactions(metaWithEvents))
+	require.Len(t, store.transactions, 1)
+
+	// check events data
+	tx, ok, _ := store.GetTransaction(txHash(1, false))
+	require.True(t, ok)
+	require.NotNil(t, tx.Events)
+	require.Equal(t, 1, len(tx.Events))
```

Comment:
use `require.Len` like you did above on L356 to get better debug logs on failures!